extensions/notebook/src/common/extensionContextHelper.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+export class ExtensionContextHelper {
```

Comment:
Why do we need this class? Is it to help with testing?

Reply:
Sorta. That + not having to plumb AppContext through to everything. See [this thread](https://teams.microsoft.com/l/message/19:6665c6ef60764ece84fc375b83d11eb8@thread.skype/1602546273325?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=a799d1d3-bb58-4048-a03c-52aa8e37f3b8&parentMessageId=1602546273325&teamName=SQL%20Tools&channelName=ADS%20Programming%20Support&createdTime=1602546273325)

Reply:
I'm not sold on this approach so if you have a better suggestion I'm open to ideas. I could also just use AppContext but that's going to take a few more changes to fix up the test + plumb it through to the session manager.

Reply:
The only other thing I can think of would be to pass through an AppContext to this class or use another one. The AppContext is basically a singleton instance already, so this static class seems fine to me.